testing/manual/calibrate.md

```diff
@@ -35,6 +34,15 @@ sampling combinations in PyCIEMSS can result in numerical instability, when this
 
 ### 4. Check results
 1. Verify that there is a loss chart available while the process is running, showing the loss value as a function of solver iterations
-2. Verify that there is a Variables chart available, showing time-series plots for each model variable based on a Simulate run with the calibrate parameter values.
-3. Verify the dataset colummn to which a given model variable is mapped appear as a dashed line in the Variables chart of the model variable
-4. Should be able to add/remove Variables charts
+2. Verify that there is a `Parameters chart` available. This should allow the user to select different model parameters (Eg beta, gamma) and will display a bar chart showing before and afer calibration values for the selected parameters.
+3. Verify that there is a `Variables chart` available, showing time-series plots for each model variable based on a Simulate run with the calibrate parameter values.
+4. Verify the dataset colummn to which a given model variable is mapped appear as a dotted line in the Variables chart of the model variable. This should have a legend with the name Observations
+5. Verify there is an `Error chart` available. This should allow the user to select different variables that were calibrated over. This will display the mean absolute error between the post forecast run and the incoming dataset.
```

Comment:
Verify that an `Error chart` is available. This should allow the user to select different calibrated variables. It will display the mean absolute error between the post-forcast run and the incoming dataset.